packages/browser/src/tracing/request.ts

```diff
@@ -374,6 +377,7 @@ export function xhrCallback(
             'server.address': host,
             [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.http.browser',
             [SEMANTIC_ATTRIBUTE_SENTRY_OP]: 'http.client',
+            body: graphqlRequest,
```

Comment:
Instead of adding this here, _after_ the span was started, in this file do:

```js
client.emit('outgoingRequestSpanStart', span, { body });
```

So the hook receives the body as second argument, and then in the integration we can use the body and add it to the span or use it otherwise. This way, this is not added to _all_ spans, as we do not want to generally capture this for every span.

Reply:
I get what you mean now! Will implement